setup.cfg

```diff
@@ -47,7 +48,7 @@ testing =
 	pytest-checkdocs >= 1.2.3
 	pytest-cov
 	pytest-enabler >= 1.0.1
-	pytest-ruff != 0.3
+	pytest-ruff >= 0.2.1
```

Comment:
Is there something in 0.2.0 that's broken? I'm an advocate of leaving dependency versions as unrestricted as possible so that end users can decide what works best for them.

Reply:
Never mind, I just saw that 0.2.0 was yanked because it's broken. I think supporting <0.2.0 might be a pain because those are basically alpha versions, so this looks good!